src/Persistence/Sql.php

```diff
@@ -436,7 +436,9 @@ public function tryLoad(Model $model, $id): ?array
      */
     public function load(Model $model, $id): array
     {
-        if (!$data = $this->tryLoad($model, $id)) {
+        $data = $this->tryLoad($model, $id);
+
```

Comment:
and no space ;-) - we load and "assert" immediatelly the loaded val...

Reply:
t is not so readable when no space. I keep it like it was originaly

Reply:
it is, sequentially readable and logically grouped